### DISCUSSION TITLE
All Settings Lost During Plugin Update

### DISCUSSION THREAD

**Discussion by WillScott73** [Cloud9Developer/Jellyfin-Newsletter-Plugin#99] [Q&A]
Is it by design that all of the plugin settings are reset to the default when the plugin auto updates?

**Comment by Cloud9Developer** [comment]
It is not. That said, I am doing some refactoring of the codebase, so maybe there was something unintentional there?

**Reply by Cloud9Developer** [reply]
However, I uninstalled, and reinstalled on my end as well and so long as the /config/plugins/configuration/<NEWSLETTERS>.xml file was there, my setting carried over.
